### DISCUSSION TITLE
ScriptReference/Unwrapping.GenerateSecondaryUVSet

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#684] [Notes]
# ScriptReference/Unwrapping.GenerateSecondaryUVSet

https://docs.unity3d.com/ScriptReference/Unwrapping.GenerateSecondaryUVSet.html

**Comment by forestrf** [comment]
Sometimes this won't generate the same UV2 as Model Import Settings' generateSecondaryUV.
